cypress/e2e/gui/testsGui.cy.js

```diff
@@ -0,0 +1,358 @@
+describe('Validate the GUI', () => {
+  context('Validate the no customers scenarios', () => {
+    beforeEach(() => {
+      const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_API_URL}?**`,
+        {
+          'customers': [],
+          'pageInfo': {
+            'currentPage': 1,
+            'totalPages': 1,
+            'totalCustomers': 50
+          }
+        }
+      ).as('getNoCustomers');
+      cy.setCookie('cookieConsent', 'accepted');
+      cy.visit('/');
+    });
+    it('The empty box image should be visible', () => {
+      cy.get('[title="image of an empty box"]').should('be.visible');
+    });
+    it('The input name should be disabled', () => {
+      cy.get('[data-testid="name"]')
+        .should('be.visible')
+        .and('have.attr', 'disabled');
+    });
+  });
+
+  context('Validate the customer missing info scenarios', () => {
+    beforeEach(() => {
+      const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`;
+      cy.intercept(
+        'GET',
+        `${CUSTOMERS_API_URL}?**`,
+        { fixture: 'customerMissingInfo.json' }
+      ).as('getCustomer');
+      cy.setCookie('cookieConsent', 'accepted');
+      cy.visit('/');
+      cy.contains('button', 'View').click();
+    });
+    it('Should not show contact info', () => {
+      cy.contains('No contact info available').should('be.visible');
+    });
+    it('Should not show address', () => {
+      cy.contains('Show address').click();
+      cy.contains('No address available').should('be.visible');
+    });
+  });
+
+  context('Validate the cookies scenarios', () => {
```

Comment:
Que tal uma descrição como a seguinte?

```suggestion
  context('Cookies consent banner', () => {
```